lib/lte_link_control/lte_lc_helpers.c

```diff
@@ -973,6 +977,8 @@ int parse_ncellmeas(const char *at_response, struct lte_lc_cells_info *cells)
 
 	err = at_parser_cmd_count_get(&parser, &count);
 	if (err) {
+		LOG_ERR("Could not get NCELLMEAS param count, "
```

Comment:
Is this possible in practice?

Reply:
Depends on where we draw the line. If modem has a bug and AT notification is somehow malformed, we could hit this error log. My principle has been that we need to protect the code from any input sent/responded by the modem, or the client through the public API.